metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ghoneycutt-nfs",
-  "version": "1.8.0",
+  "version": "1.8.1",
```

Comment:
Can you please unstage this and rebase your commit? It is good practice to not bump versions in a commit as the upstream maintainer will manage this as they might release other code before this.